src/helper/ssg/index.test.tsx

```diff
@@ -336,6 +366,30 @@ describe('saveContentToFiles function', () => {
     expect(fsMock.writeFile).toHaveBeenCalledWith('static/bravo.text/index.html', 'Bravo Page')
   })
 
+  describe('should correctly create binary files with the right content', () => {
+    it('for application/gzip', async () => {
+      await saveContentToFiles(htmlMap, fsMock, './static')
+
+      expect(fsMock.writeFile).toHaveBeenCalledWith('static/bravo/index.tar.gz', expect.anything())
```

Comment:
Perhaps this line is useless

Reply:
Does that mean that there is a following test?

https://github.com/honojs/hono/blob/f6957ee1b2b83588a9ba472a91df1f48ebcf1480/src/helper/ssg/index.test.tsx#L374-L378

I think we can leave this line in, since the other files are tested with `toHaveBeenCalledWith` in the same way.

Reply:
OK, I've found that the followings is more unnecessary. This is the remains of my investigation

Removed in 7b17acbbc0be407621af4aaa340f3fdd25fd5dfd

Reply:
hmm... we can merge again into `"should correctly create files with the right content and paths"` test